pkg/kubectl/explain/field_lookup_test.go

```diff
@@ -69,9 +69,9 @@ func TestFindField(t *testing.T) {
 			gotPath = path.GetPath().String()
 		}
 
-		if gotErr != test.err && gotPath != test.expectedPath {
-			t.Errorf("LookupSchemaForField(schema, %v) = (%v, %v), expected (%s, %v)",
-				test.path, gotErr, gotPath, test.expectedPath, test.err)
+		if gotErr != test.err && gotPath == test.expectedPath {
+			t.Errorf("LookupSchemaForField(schema, %v) = (%v), expected (%v)",
+				test.path, gotErr, test.err)
```

Comment:
I'd like to test both values and print both values in the same error.

Reply:
With a single if, the message gets little unreadable. For example,
`LookupSchemaForField(schema, [field1 what?]) = (field "what?" does not exist, ), expected (, field “X” does not exist)`
vs proposed
`LookupSchemaForField(schema, [field1 what?]) = (field "what?" does not exist), expected (field “X” does not exist)` if the value of “err” is changed by replacing “what?” with “X”.
Let me update another commit with two if loops and see if that makes more sense? Thanks!

Reply:
I don't believe I like that proposed better than the current. `LookupSchemaForField(schema, [field1 what?])` returns *two* values, it's very weird to show only one. 

When I see `(field "what?" does not exist, ), expected (, field “X” does not exist)`, I know exactly what's wrong. Maybe we could name the parameters:

```
LookupSchemaForField(schema, [field1 what?]) = (path:"field1.what?", err:""), expected (path:"", err:"field “what?” does not exist")```

Reply:
Sure, that's better. Thanks!